zschema/keys.py

```diff
@@ -30,6 +30,62 @@ def __cmp__(self, other):
             return cmp(int(self.port), int(other.port))
 
 
+def left_merge(lhs, rhs):
```

Comment:
This should probably should be private (i.e., `_left_merge`).

Reply:
Makes sense, though I know we are merging elsewhere, so it might make sense to unify all of those. But for now I'll just make it private and add a TODO.